pkg/oc/cli/admin/release/new.go

```diff
@@ -538,6 +549,32 @@ func (o *NewOptions) Run() error {
 		is.Annotations = make(map[string]string)
 	}
 
+	if o.CoreOSURL != "" {
+		coreosBuild, err := GetCoreOSBuild(context.TODO(), o.CoreOSURL, o.CoreOSVersion)
+		if err != nil {
+			return err
+		}
+		fmt.Fprintf(o.Out, "Using CoreOS build %s\n", coreosBuild.BuildID)
+		digestedImage := fmt.Sprintf("%s@%s", coreosBuild.OSContainer.Image, coreosBuild.OSContainer.Digest)
+		if digestedImage == "@" {
+			return fmt.Errorf("No oscontainer in CoreOS build")
```

Comment:
Why not do an explicit e.g. `if coreosBuild.OSContainer.Image == "" || coreosBuild.OSContainer.Digest == "" { ... }`? (This also catches the case where just one of the two is somehow missing.)